### ISSUE TITLE
A couple issues on Linux

### ISSUE THREAD

**Issue by MetalAZ** [amir9480/vscode-laravel-extra-intellisense#2]
When I installed the extension through VSCode it didn't work at all. Developer Tools showed a case issue with the routeProvider.js file. Linux is case sensitive and the R needed to be uppercase (same issue with viewProvider.js). I downloaded the repo and did a compile and the problem was gone -- it was the correct case.

But that left one other issue. Linux (and I think macOS) uses Bash shell and when there are $ signs in a double quoted string Bash treats it as a variable, which breaks running the PHP command. I was able to resolve the issue by updating the helpers.ts file:

1. I imported 'os':
`import * as os from 'os';`

2. I changed the runPhp function to:
```
	static runPhp(code: string): string {
		if (fs.existsSync(Helpers.projectPath("vendor/autoload.php")) && fs.existsSync(Helpers.projectPath("bootstrap/app.php"))) {

			var command = "php -r \"define('LARAVEL_START', microtime(true));require_once '" + Helpers.projectPath("vendor/autoload.php") + "'; require_once '" + Helpers.projectPath("bootstrap/app.php") + "';$kernel = $app->make(Illuminate\\Contracts\\Http\\Kernel::class);$response = $kernel->handle(     $request = Illuminate\\Http\\Request::capture() );" + code + "\"";;

			if (['linux', 'openbsd', 'sunos', 'darwin'].findIndex(x => x === os.platform()) > -1) { // aix, darwin, linux, openbsd, sunos, win32
				command = command.replace(/\$/g, "\\$");
			}

			return cp.execSync(command).toString();
		}
		return "";
	}
```
Basically I'm escaping the $ signs if the operating system matches one in the list.

**Comment by amir9480** [comment]
Thanks for report.
Issue fixed in version 0.1.0.
